src/main.py

```
@@ -0,0 +1,266 @@
+    if not settings.OPENAI_API_KEY:
+        raise HTTPException(status_code=400, detail="OPENAI_API_KEY not configured")
+
+    # Initialize services
+    openai_client: OpenAIImageClient | None = None
+    try:
+        openai_client = OpenAIImageClient(api_key=settings.OPENAI_API_KEY)
+        orchestrator = GenAIOrchestrator(openai_client=openai_client)
+        processor = ImageProcessor()
+        storage = StorageManager(base_path=Path("outputs"))
+    except Exception as e:  # pragma: no cover - defensive
+        logger.exception("Failed to initialize services", e)
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)